apps/emloyees-management/src/app/app.module.ts

```diff
@@ -1,11 +1,23 @@
 import { NgModule } from '@angular/core';
+import { AngularFireModule } from '@angular/fire';
+import { AngularFireAuthModule } from '@angular/fire/auth';
 import { BrowserModule } from '@angular/platform-browser';
-
+import { EmployeeAppFeatureAuthModule } from '@dashasorg/employee-app/feature-auth';
+import { environment } from '../environments/environment';
+import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { HomeComponent } from './home/home.component';
+
 
 @NgModule({
-  declarations: [AppComponent],
-  imports: [BrowserModule],
+  declarations: [AppComponent, HomeComponent],
+  imports: [
+    BrowserModule,
+    AppRoutingModule,
+    AngularFireModule.initializeApp(environment.firebase),
+    AngularFireAuthModule,
+    EmployeeAppFeatureAuthModule
```

Comment:
Why you import this module to  App Module? it should be available throughout the routing

Reply:
fixed